presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1728,4 +1792,11 @@ ThriftMetastoreClient createMetastoreClient()
         T callOn(ThriftMetastoreClient client)
                 throws TException;
     }
+
+    @FunctionalInterface
+    private interface Call1<A>
```

Comment:
Nice name.

Reply:
thanks. or, is it a request for a change?

Reply:
I would like to request a change. But I couldn't find anything better to suggest.

Reply:
i used this name over `CallWithArgument` because one day i will need to interface for call with two arguments...

anyway, this is private to this class and impl'd as lambda, so don't think it's a big deal